masq_lib/src/utils.rs

```diff
@@ -119,6 +119,13 @@ macro_rules! short_writeln {
     };
 }
 
+#[macro_export]
+macro_rules! intentionally_blank {
+    () => {
+        panic!("unrequired method of a trait; left blank here")
```

Comment:
Maybe something like "Required method left unimplemented: should never be called."